exec/exploratory_plots.R

```diff
@@ -100,6 +100,12 @@ option_list <- list(
     metavar = "string",
     help = "A valid R palette name.",
     default = "Set1"
+  ),
+  make_option(
+    c("-l", "--log2_assays"),
```

Comment:
I think we might want to allow users to specify which assay(s) should be logged, rather than applying a blanket boolean. So this should be a comma-separated string of assay names or numbers.

Reply:
Something like this?